server/src/messages/messages.controller.ts

```diff
@@ -10,14 +10,17 @@ import { DeleteResult } from 'typeorm';
 export class MessagesController {
   constructor(private readonly messagesService: MessagesService) {}
 
-  @UseGuards(JwtAuthGuard)
+  // @UseGuards(JwtAuthGuard)
   @Post()
   async create(@Body() createMessageDto: CreateMessageDto): Promise<Message> {
-    console.log(createMessageDto);
     return this.messagesService.create(createMessageDto);
   }
 
-  @UseGuards(JwtAuthGuard)
+  // TODO: reinstate UseGuards here and in the Delete action when
+  // ready - currently these are needed to add data to test the
+  // MessageInboxView component
+
+  // @UseGuards(JwtAuthGuard)
```

Comment:
actually, remove these altogether
i'm working on something that'll replace these anyways
(will use cookie guard in the future https://github.com/Nonprofit-Exchange-Hub/web-app/pull/91/files)